components/21-atoms/table/table.functions.js

```diff
@@ -33,7 +33,7 @@
         // If not remove the tab focus.
         element.removeAttribute('tabindex');
       }
-      else {
+      else if (caption) {
```

Comment:
Caption is required.
This makes me a bit nervous.

Reply:
Semantically speaking, I don't think it's required. (See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table)). Also: A JS error is thrown when it's not there, stopping all JS execution and that feels kind of drastic to me. 

As an alternative: we could log a warning in the console to let people know they SHOULD use a caption? 🤷‍♀️

Reply:
No, you are correct.
Advisable, not required.

https://webaim.org/techniques/tables/data

Keep up the good work :+1: